README.md

```diff
@@ -90,47 +90,77 @@ Members have different degrees of love for each of the communities that they par
 
 ### Measuring love
 
-In the Orbit Model, a member's love is inferred by the quantity and quality of activities they've completed.
-To calculate love, start by counting all the activities that a member has done within a recent time period. For example:
+In the Orbit Model, a member's love is inferred by the quantity, quality, and recency of activities they've completed. In other words, for each member, you want to understand what they've contributed, the relative importance of each contribution, and when the contribution happened. 
+
+Start by making a list of all the activities that are important for your community. For example:
 
 - Attended our conference
 - Spoke at a meetup
 - Opened a pull request on GitHub
 - Answered a forum question
 
-It's important to track when members complete these activities so you can do the math. Each activity is given a score of how much engagement it represents. We recommend using a simple 1-10 scale.
+Then consider the relative importance of each kind of activity you track. We recommend using a simple 1-3 scale.
+
+> "Organizing a community meetup takes a lot of time and effort, so we give it an activity weight of 3. Just attending a meetup is a 1, but publishing a summary may count for 2."
 
-> "Organizing a community meetup takes a lot of time and effort, so we give it an activity score of 8/10. Just attending a meetup is a 3/10, but publishing a summary counts for 2 more points."
+Since all activities default to a weight of 1, weighting activities is entirely optional. Some communities will require and welcome the complexity, while others will be just fine with the defauult weights. 
```

Comment:
typo: _default_

Reply:
Maybe we could recommend starting with all weights at 1, and adding weights to each activity once the need for finer-grained reporting/analysis arises?